pascalbakker/OpenComic
# Fixed bugs and updated formatting

# Description

There was a bug where large files would not open in OpenComic. This was due to passing all images as an argument in a list to a new asynchronous child when extracting 7z, resulting in ENAMETOOLONG error. This was resolved by making the function no longer asynchronous, as it was not required. 

# Changelog

- Updated formatting in reading.js for more compactness
- Updated formatting in file-manager.js for more compactness
- Fixed bug with opening large files
- Moved save image logic to separate for better readability

# Testing

1. Build project
2. Open a large file that is 450+ images large
3. See that it no longer infinitely stalls, and actually opens the image
